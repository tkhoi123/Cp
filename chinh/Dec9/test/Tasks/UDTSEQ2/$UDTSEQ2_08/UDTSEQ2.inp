3 650
3 3 -23579
2 2 23910
2 3 -5942
2 3 -7120
3 3 -16913
3 3 8793
1 1 14849
1 2 -6008
1 2 18790
2 3 -2031
3 3 -18437
1 2 -11447
3 3 11269
3 3 -17005
2 2 -17944
1 3 29908
1 1 24122
2 3 17642
3 3 -11845
2 2 17171
3 3 21505
2 3 -22704
2 2 -7392
1 3 3749
3 3 5043
2 3 21565
1 1 22149
2 2 -6186
1 3 10314
1 1 -3191
1 2 -5346
2 2 22191
1 3 -8153
2 3 -8252
1 1 11401
2 3 -22072
2 2 25049
3 3 -15128
3 3 -24523
2 3 26530
2 2 13384
1 2 -4013
1 1 16595
2 2 -21436
1 1 1101
1 3 -30494
1 2 -16414
3 3 -13289
1 2 3219
2 3 -25432
1 1 -5004
1 1 -15711
2 3 -22327
1 2 9370
2 2 20236
1 3 -8629
1 3 15614
1 2 16601
2 2 27489
1 3 30958
3 3 -7146
1 2 -24431
1 1 26196
1 1 -29155
3 3 -11015
1 1 7793
2 3 31783
1 2 31750
1 1 -12479
1 1 -194
2 3 16607
3 3 28696
1 1 -6957
2 3 29518
3 3 6167
2 3 658
2 3 -1246
3 3 29226
1 1 -27654
1 3 24310
1 3 -5193
1 2 -19065
1 2 -4553
1 1 -11205
1 2 17006
2 2 -11811
1 1 9103
2 3 -4640
1 3 6104
1 3 -8701
3 3 28323
1 3 30119
1 1 19670
2 2 1137
2 2 -4928
2 3 -32697
1 1 2070
2 3 -8076
1 3 -4534
1 1 20800
3 3 -31016
2 3 -15712
1 3 -32018
1 3 -12369
1 1 -27067
1 1 -27036
2 2 -24138
2 2 -14126
1 1 -4186
1 1 -2660
3 3 22620
2 3 -15554
2 2 -2817
2 3 -21353
2 3 -15498
2 3 -6531
2 2 21648
1 3 15709
3 3 8166
3 3 19485
2 2 12838
2 3 -31323
2 2 1941
2 2 14982
3 3 25754
2 2 -11555
1 3 31076
1 2 15123
2 2 7199
1 2 29030
3 3 -31420
1 1 -12574
1 3 24647
2 3 -2873
2 3 20123
3 3 28717
1 1 -28622
2 3 14069
2 2 -24474
1 3 -29147
3 3 -19430
2 2 31903
1 1 -23906
2 3 10790
1 2 -17345
1 1 -23819
1 2 17672
2 2 17183
2 2 -4855
1 3 31265
1 1 -854
1 3 24762
3 3 -12291
2 2 25223
3 3 10557
1 2 -14220
2 3 -32292
1 3 13970
3 3 12124
2 2 9552
2 3 24595
1 2 12875
2 3 1179
2 2 8224
1 3 3507
2 3 -22320
3 3 -8690
1 1 21944
1 1 -4507
3 3 -15511
2 3 -26380
3 3 -779
3 3 24156
1 3 1516
2 2 -3233
2 3 -31539
1 3 22403
1 1 -8763
2 3 -1843
1 3 -22212
2 2 -23220
1 3 31944
1 3 -31091
1 1 -14595
2 2 -3772
1 2 15422
2 2 -18459
1 2 18678
1 1 32119
3 3 -24906
2 3 -22580
2 2 -24604
3 3 19786
2 2 -17602
2 3 16546
2 3 25901
1 2 12984
1 2 -10275
2 3 13950
2 2 271
1 1 -26734
2 2 -19668
1 1 -13492
3 3 -30555
1 1 -8729
1 1 3470
2 3 -14079
2 2 -16820
3 3 31014
1 3 -5573
2 2 -20392
2 3 -13926
2 2 15634
2 3 -11416
2 3 -28421
3 3 -20791
1 2 -27358
1 2 1236
2 2 17775
3 3 17794
1 2 14140
1 1 -16144
2 2 -6604
3 3 -29816
2 2 -26682
1 2 30666
1 1 -17630
2 3 -1121
1 1 -6297
3 3 -30569
1 1 7080
3 3 13532
1 3 3529
2 3 -31482
2 3 -20101
2 2 -27289
1 2 20387
1 3 12174
1 2 4200
2 2 -4635
2 3 -20579
1 3 10123
2 3 24747
1 3 -295
1 1 11282
1 1 4713
1 1 -8172
3 3 -9763
1 2 -12686
1 3 27517
2 3 12872
2 2 -11948
2 2 27777
2 3 14447
1 2 -24191
1 1 29114
1 2 -13704
2 3 -11106
2 2 -25208
2 3 -30498
1 3 2000
1 1 -11520
1 3 22194
1 1 -15470
1 2 -21248
1 3 -114
2 2 -2911
1 1 -29533
2 3 -5165
1 1 22836
2 2 24549
1 1 6294
3 3 -8785
1 2 29698
2 3 -15636
2 3 31123
2 3 -22244
1 2 -28721
2 2 5200
3 3 -7023
1 2 -8797
2 2 -30816
1 2 -11473
2 2 14973
1 2 -2886
1 1 17632
3 3 28579
1 2 27533
1 2 -14747
1 3 31753
2 3 8400
1 3 29931
1 1 10796
1 1 1689
2 2 4398
1 3 -23024
1 1 -21114
2 3 -12232
1 1 -15211
2 3 5712
2 2 -21840
1 2 30627
2 2 -18120
2 2 -11473
2 3 9625
1 3 -4400
1 1 -27970
1 3 14975
3 3 32429
1 2 -14969
1 2 -6405
1 3 10966
1 2 20166
2 2 20398
1 1 4077
1 1 -25795
2 3 11863
2 3 31721
2 2 17604
1 3 3743
1 3 -16515
1 3 -13680
2 3 17834
1 1 32252
1 1 18068
1 3 10308
2 3 20930
1 3 2366
1 1 16753
2 2 -890
1 3 18799
2 3 20153
2 3 -27247
1 1 28505
1 3 -9726
2 2 17719
2 2 -27400
2 3 6013
1 2 12307
2 2 27363
2 3 2256
2 2 6570
1 3 -20211
1 2 3877
3 3 -5235
2 2 8958
2 3 15573
1 1 1013
2 2 14216
2 3 -15965
2 3 -414
1 1 13077
3 3 -21359
2 3 -15958
1 1 -14019
1 3 5311
1 2 19906
1 1 -16140
2 2 -2318
1 3 30568
1 1 -31824
1 2 -20393
1 2 -6939
1 3 -21046
2 3 -21505
2 2 31561
1 1 1402
1 3 24588
1 2 311
2 2 9106
2 2 28883
3 3 -32726
1 2 1784
2 2 11939
1 1 -9980
1 2 29462
2 3 21000
1 3 22289
2 2 23235
1 2 14649
1 2 -18476
2 3 -13206
2 3 -7523
1 2 18090
3 3 19447
2 3 31136
1 2 28598
3 3 30450
1 2 12883
2 2 12014
2 2 -19238
1 2 -26632
3 3 -12014
2 3 31051
2 2 20289
1 1 -8748
2 3 23255
2 2 19985
2 2 -30871
1 2 -18770
2 2 -28538
1 2 -24499
1 1 6864
2 2 -15856
2 3 29344
3 3 -10605
1 2 -15922
1 2 15645
2 2 -28299
1 1 -21675
2 3 31708
3 3 11936
2 3 9905
2 3 -10651
1 1 -23537
3 3 -10637
1 1 -28183
1 2 -11220
2 2 23474
1 2 10112
1 3 -32319
3 3 19465
2 2 -29844
3 3 18766
1 3 22062
1 2 -29946
1 1 17690
1 1 27319
3 3 4845
2 2 5945
1 3 -26224
1 3 8324
1 2 15404
2 2 -11180
2 3 22837
1 2 -19594
2 2 31199
2 2 12342
1 3 8243
2 2 -7311
1 3 8125
3 3 11293
1 3 -5920
2 3 -5180
1 1 -31007
2 2 -26468
1 3 -473
1 2 4416
3 3 -17404
2 2 -30715
2 2 9410
1 2 13144
3 3 25568
3 3 13842
2 3 10405
1 3 5813
1 1 -19601
1 2 -15738
2 2 -16504
1 3 -20480
1 2 14041
2 3 -25516
2 2 -18365
3 3 -10546
2 2 -30174
1 3 2419
2 2 -10799
1 2 -7235
1 1 -32505
2 3 28863
1 2 -23675
1 3 -8155
1 2 7075
1 2 -16322
1 3 -27902
1 3 30929
3 3 -7833
3 3 31474
1 1 2955
1 3 18503
1 3 -4967
2 2 -4381
1 2 32554
1 3 21240
2 2 8960
1 2 2066
1 2 9095
2 2 14527
1 3 24253
1 1 -18192
1 3 3102
1 3 -30134
1 1 -24167
3 3 -21907
2 2 -22186
1 1 -10436
2 3 31026
1 1 -3185
1 2 -7141
1 1 3858
2 2 -15420
1 2 -13012
1 3 4295
3 3 -6210
1 3 28286
1 2 -29417
2 2 6640
2 2 17508
3 3 -4111
2 2 10806
1 2 -1277
1 2 9328
3 3 9800
1 2 -900
3 3 -22781
2 3 -16291
2 2 10581
1 1 12109
1 2 -17544
1 1 -27170
1 3 30929
1 3 -16079
2 2 -1924
1 2 -17656
3 3 -16893
3 3 14512
2 3 -11787
3 3 6786
1 1 3808
1 2 -23273
2 2 -14522
1 3 16116
1 1 13079
2 3 7819
3 3 -7555
1 2 -19639
1 1 -31142
1 1 -27720
2 2 -10297
2 2 26586
1 1 -25418
2 2 -26805
2 2 -17071
1 2 1981
1 2 -29250
2 2 30336
1 1 24578
2 3 10757
2 2 21588
2 3 3236
1 1 5168
1 1 16815
3 3 -1178
1 1 -20782
3 3 7001
2 3 13853
2 2 -22402
1 1 12471
2 2 20973
2 3 30509
1 3 -11879
1 1 29372
1 3 2008
1 1 8273
1 3 -10184
1 3 25075
3 3 10343
1 3 17104
2 2 17948
2 3 -2682
3 3 7390
1 1 -22368
2 2 27742
1 2 -2042
1 2 18524
1 1 9796
3 3 -3387
3 3 10562
1 1 4617
1 2 -30261
3 3 -32463
1 1 -24165
2 2 20925
3 3 -29712
1 2 7030
1 1 -4379
1 2 -31583
2 3 -27574
2 2 14936
3 3 6164
1 2 -8074
2 2 -10038
2 2 -30506
1 1 2450
3 3 -6624
1 1 -21840
3 3 1061
2 3 -2548
1 3 6923
3 3 -14464
2 2 6173
1 2 19019
2 3 -199
3 3 -21886
1 2 -3477
2 2 -6923
1 2 -10952
2 3 14974
2 3 3154
2 3 16001
1 1 -7838
3 3 23715
2 2 -23306
2 3 -9588
2 2 23892
1 2 -29104
3 3 31652
3 3 -27943
2 3 11381
3 3 1935
1 2 9065
2 2 13341
1 1 27739
1 3 -5025
2 2 6101
1 2 8642
1 2 -31243
1 1 -16116
3 3 -18188
3 3 -10036
3 3 -19081
1 2 -14254
1 1 -28364
2 2 -18127
3 3 24920
1 2 -23641
3 3 -13824
3 3 25595
3 3 14
2 3 -8280
3 3 -24935
1 3 21960
1 3 -10752
1 2 -20344
1 3 -8455
2 3 28588
2 2 17731
1 1 -22289
1 1 -32527
1 3 9919
