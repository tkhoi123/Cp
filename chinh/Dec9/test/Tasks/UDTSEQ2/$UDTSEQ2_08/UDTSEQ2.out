999770657 524771 1298341 